src/Webfactory/Slimdump/Database/Dumper.php

```diff
@@ -92,40 +90,12 @@ public function dumpSchema($table, Connection $db, $keepAutoIncrement = true, bo
      */
     public function dumpTriggers(Connection $db, $tableName, $level = Table::DEFINER_NO_DEFINER)
     {
-        $triggers = $db->fetchAll(sprintf('SHOW TRIGGERS LIKE %s', $db->quote($tableName)));
-
-        if (!$triggers) {
-            return;
-        }
-
-        $this->output->writeln("-- BEGIN TRIGGERS $tableName", OutputInterface::OUTPUT_RAW);
-
-        $this->output->writeln("DELIMITER ;;\n");
-
-        foreach ($triggers as $row) {
-            $createTriggerCommand = $db->fetchColumn("SHOW CREATE TRIGGER `{$row['Trigger']}`", [], 2);
-
-            if (Table::DEFINER_NO_DEFINER === $level) {
-                $createTriggerCommand = preg_replace('/DEFINER=`[^`]*`@`[^`]*` /', '', $createTriggerCommand);
-            }
-
-            $this->output->writeln($createTriggerCommand.";;\n", OutputInterface::OUTPUT_RAW);
-        }
-
-        $this->output->writeln('DELIMITER ;');
+        $this->sqlDumper->dumpTriggers($db, $tableName, $level);
```

Comment:
Triggers and Views are two concepts that won't make sense for CSV exports.

Would it be possible to have just one method on the `SqlDumper` (like `beginTable($tablename)`?), and that would dump the schema structure and triggers (for tables) or the view creation statement (for views)?